service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,7 +80,10 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
         Document document = uploadDocumentService.uploadPDF(userId, authorization, pdf, buildFileName(caseDetails));
 
+        ZonedDateTime zonedDateTime = ZonedDateTime.now(ZoneId.of("Europe/London"));
+
         Map<String, Object> data = caseDetails.getData();
+        data.put("dateSubmitted", DateTimeFormatter.ISO_LOCAL_DATE_TIME.format(zonedDateTime));
```

Comment:
Since you switched to `ZonedDateTime` we should use `ISO_OFFSET_DATE_TIME` instead of `ISO_LOCAL_DATE_TIME`. It will add offset to serialised string which I hope CCD can handle properly.